gordo_components/cli/client.py

```diff
@@ -29,7 +29,6 @@
 @click.option("--host", help="The host the server is running on", default="localhost")
 @click.option("--port", help="Port the server is running on", default=443)
 @click.option("--scheme", help="tcp/http/https", default="https")
-@click.option("--gordo-version", help="Version of gordo", default="v0")
```

Comment:
Why remove this?

Reply:
because, as I saw it, running a given version of gordo client against a different major version would intrinsically be a bad idea. Maybe I am wrong in that assumption.